$OpenBSD: patch-configure,v 1.1 2013/04/13 10:06:08 jasper Exp $

XXX: No configure switch to toggle this, so set it manually for now.

--- configure.orig	Sat Apr 13 11:28:08 2013
+++ configure	Sat Apr 13 11:28:17 2013
@@ -13756,7 +13756,7 @@ GCR_REQUIRED=3.5.5
 AVAHI_REQUIRED=0.6.22
 
 # Set to 'no' to build with the WebKit1 backend
-with_webkit2=yes
+with_webkit2=no
 
 if test "x$with_webkit2" != xno; then
    WEBKIT_GTK_PC_NAME=webkit2gtk-3.0
